### PR TITLE
GITHUB-37: migrate to new CDP client

### PR DESCRIPTION
**Pull request by sskorol**
- Migrated client and protocol to the latest version from [this fork](https://github.com/sskorol/chrome-devtools-java-client.git)
- Migrated to JDK 17
- Updated outdated dependencies
- Fixed broken tests due to protocol updates
- Simplified WS connection with a new [API](https://github.com/sskorol/chrome-devtools-java-client/blob/master/cdt-java-client/src/main/java/io/github/sskorol/cdt/services/ChromeDevToolsService.java#L115-L144)

@vania-pooh fiy

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -9,14 +9,22 @@
...
+        <java.version>17</java.version>
+        <selenium.version>4.1.4</selenium.version>
+        <cdt-java-client.version>5.0.0</cdt-java-client.version>
+        <junit-jupiter.version>5.8.2</junit-jupiter.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <slf4j-log4j12.version>1.7.30</slf4j-log4j12.version>
+        <slf4j-log4j12.version>2.0.0-alpha7</slf4j-log4j12.version>
     </properties>
 
+    <repositories>
```
Comment (vania-pooh) [inline pom.xml]: Do we need this? Seems to be useless.

**Review by vania-pooh — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (sskorol) [reply pom.xml]: Right, I initially tested it from within local repo. Will remove.

Comment (sskorol) [reply pom.xml]: Done

**Review by vania-pooh — APPROVED** [review]

**Comment by vania-pooh** [comment]
@sskorol thank you for contribution!

**Pull request merged**